php/cli/Dockerfile

```diff
@@ -30,5 +31,5 @@ RUN set -eux \
 ENV COMPOSER_ALLOW_SUPERUSER 1
 ENV COMPOSER_HOME /tmp/composer
 COPY --from=composer:1 /usr/bin/composer /usr/bin/composer1
-COPY --from=composer:2 /usr/bin/composer /usr/bin/composer2
-COPY --from=composer:2.2 /usr/bin/composer /usr/bin/composer2lts
+COPY --from=composer:${COMPOSER_VERSION} /usr/bin/composer /usr/bin/composer2
+COPY --from=composer:lts /usr/bin/composer /usr/bin/composer2lts
```

Comment:
I can see a future where lts is not 2 and everything breaks

Reply:
@navarr reverted as it was.